ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__arm64-stable-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -0,0 +1,342 @@
+base_images:
+  ansible:
+    name: "4.14"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.14"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.15"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.14"
+  tools:
+    name: "4.14"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.14"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  arm64-latest:
+    prerelease:
+      architecture: arm64
+      product: ocp
+      version_bounds:
+        lower: 4.14.0-0
+        upper: 4.15.0-0
+  arm64-target:
+    prerelease:
+      architecture: arm64
+      product: ocp
+      version_bounds:
+        lower: 4.15.0-0
+        upper: 4.16.0-0
+  latest:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.14.0-0
+        upper: 4.15.0-0
+  target:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.15.0-0
+        upper: 4.16.0-0
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: aws-ipi-byo-iam-role-f28
```

Comment:
This yaml should be copied from `openshift-openshift-tests-private-release-4.14__arm64-stable-4.14-upgrade-from-stable-4.13.yaml`, then update the version setting, but not copied from nightly-upgrade-from-stable, right?

Reply:
It depends on how many profiles do we want to cover in stable to stable upgrade tests, full profile coverage or partial profile coverage ?
I'm fine with either one. Which one do you prefer ?

Reply:
I prefer partial profile coverage, the stable to stable upgrade tests are selected on some specific install profiles.

Reply:
Ack